src/bundle/Resources/public/js/scripts/button.content.edit.js

```diff
@@ -34,6 +35,10 @@
             $('#version-draft-conflict-modal').modal('show');
         };
 
+        const showError = (error) => {
```

Comment:
No empty line between `const`

Reply:
Please make it a one liner: `const showError = (error) => showErrorNotification(error);`

Reply:
I have a better idea. Why not call `showErrorNotification` directly in `then()`?

Reply:
@dew326 @sunpietro is it ok now?